components/monitoring/prometheus/base/monitoringstack/monitoringstack.yaml

```diff
@@ -100,6 +100,11 @@ spec:
         - '{__name__="kube_pod_status_phase", namespace!~".*-tenant|openshift-.*|kube-.*"}'
         - '{__name__="kube_persistentvolume_status_phase", namespace!~".*-tenant|openshift-.*|kube-.*"}'
         - '{__name__="kube_resourcequota", namespace!~".*-tenant|openshift-.*|kube-.*"}'
+        - '{__name__="kube_statefulset_status_replicas_ready", namespace!~".*-tenant|openshift-.*|kube-.*"}'
+        - '{__name__="kube_statefulset_replicas", namespace!~".*-tenant|openshift-.*|kube-.*"}'
+        - '{__name__="openshift_route_status", namespace!~".*-tenant|openshift-.*|kube-.*"}'
+        - '{__name__="kube_deployment_status_replicas_ready", namespace!~".*-tenant|openshift-.*|kube-.*"}'
+        - '{__name__="kube_deployment_spec_replicas", namespace!~".*-tenant|openshift-.*|kube-.*"}'
```

Comment:
@jparsai Do you have any notion for how many time series we might expect from those metrics? The concern here is that those metrics appear to be of unbound cardinality, meaning that this can easily explode and require more storage than RHOBS provide us.

Also, can we limit the scope of what is being forwarded even further? E.g. exact namespace(s).

Reply:
3 out of 5 metrics have 1 time series each and 2 have 4 time series each. 
Yes scope can be reduced, all these metrics need to be collected from single namespace i.e. `gitops-service-argocd`. Is this how I can fix the namespace?
`- '{__name__="kube_deployment_spec_replicas", namespace="gitops-service-argocd"}'`

Reply:
`{__name__="kube_deployment_spec_replicas", namespace="gitops-service-argocd"}` looks good. To confirm, you can try querying for it inside the OpenShift web console (Observe -> Metrics IIRC) and see if it returns what you expect.

Reply:
It is updated, PTAL.